src/comparator/service_comparator.py

```diff
@@ -31,61 +33,86 @@ def compare(self):
-                FindingContainer.addFinding(FindingCategory.METHOD_RESPONSE_TYPE_CHANGE, "", msg, True)
+                FindingContainer.addFinding(
+                    FindingCategory.METHOD_RESPONSE_TYPE_CHANGE, "", msg, True)
             # 6.5 The request streaming state of an RPC method is changed.
-            if method_original.client_streaming != method_update.client_streaming:
-                msg = 'The request streaming type of method {} is changed'.format(name)
-                FindingContainer.addFinding(FindingCategory.METHOD_CLIENT_STREAMING_CHANGE, "", msg, True)
+            if method_original.client_streaming != \
+               method_update.client_streaming:
+                msg = 'The request streaming type of method {} is changed'.format(
+                    name)
+                FindingContainer.addFinding(
+                    FindingCategory.METHOD_CLIENT_STREAMING_CHANGE, "", msg, True)
             # 6.6 The response streaming state of an RPC method is changed.
             if method_original.server_streaming != method_update.server_streaming:
-                msg = 'The response streaming type of method {} is changed'.format(name)
-                FindingContainer.addFinding(FindingCategory.METHOD_SERVER_STREAMING_CHANGE, "", msg, True)
+                msg = 'The response streaming type of method {} is changed'.format(
+                    name)
+                FindingContainer.addFinding(
+                    FindingCategory.METHOD_SERVER_STREAMING_CHANGE,
+                    "",
+                    msg,
+                    True
+                )
             # 6.7 The paginated response of an RPC method is changed.
             if self._isPaginatedResponse(method_original) != self._isPaginatedResponse(method_update):
-                msg = 'The paginated response of method {} is changed'.format(name)
-                FindingContainer.addFinding(FindingCategory.METHOD_PAGINATED_RESPONSE_CHANGE, "", msg, True)
+                msg = 'The paginated response of method {} is changed'.format(
+                    name)
+                FindingContainer.addFinding(
+                    FindingCategory.METHOD_PAGINATED_RESPONSE_CHANGE,
+                    "",
+                    msg,
+                    True
+                )
 
     def _isPaginatedResponse(
-        self, 
-        method: MethodDescriptorProto) -> bool:
+        self,
+        method: MethodDescriptorProto
+    ) -> bool:
         # (AIP 158) The response must not be a streaming response.
         if method.server_streaming:
             return False
         responseMsg = method.output_type
         # API must provide a `string next_page_token` field.
-        if ('next_page_token' not in responseMsg.fields_by_name) or responseMsg.fields_by_name['next_page_token'].type != 9:
+        if ('next_page_token' not in responseMsg.fields_by_name) or \
+                responseMsg.fields_by_name['next_page_token'].type != 9:
```

Comment:
Very minor nit: the `\` isn't necessary if you wrap the entire conditional in parentheses.

Reply:
Noted, this is helpful! Added parentheses to remove the `/`.